include/CppLinuxSerial/SerialPort.hpp

```diff
@@ -173,6 +173,10 @@ namespace mn {
             /// \throws		CppLinuxSerial::Exception if state != OPEN.
             int32_t Available();
 
+            /// \brief          Use to get the state of the serial port
+            /// \returns        The state of the serial port
+            State getState();
```

Comment:
@SibertEnovates could you please capitalize the "g" in getState() please? (to follow convention).

Reply:
Done